### ISSUE TITLE
mainline-alpine latest image has vulnerabilities

### ISSUE THREAD

**Issue by yanshein** [nginxinc/docker-nginx#567]
Following vulnerabilities were detected in latest mainline-alpine image:
CVE-2021-36159
CVE-2021-22922
CVE-2021-22923
CVE-2021-22924
CVE-2021-22925

My apology if there is an issue already opened for it. TIA.

**Comment by JfrogPrasanna** [comment]
We are also suffering from these vulnerabilities - pls note CVE-2021-36159 is critical and a quick patch to that extent will be appreciated.

**Comment by yosifkit** [comment]
Have you pulled the latest images? https://github.com/docker-library/official-images/pull/10674 was done 11 days ago (and dependent official-images rebuilt) and explicitly covers CVE-2021-36159.

Also, there aren't any updated packages available, so there is little that can be done:
```console
$ docker pull nginx:mainline-alpine
mainline-alpine: Pulling from library/nginx
29291e31a76a: Already exists 
e82f830de071: Pull complete 
d7c9fa7589ae: Pull complete 
3c1eaf69ff49: Pull complete 
bf2b3ee132db: Pull complete 
9a6ac07b84eb: Pull complete 
Digest: sha256:bead42240255ae1485653a956ef41c9e458eb077fcb6dc664cbc3aa9701a05ce
Status: Downloaded newer image for nginx:mainline-alpine
docker.io/library/nginx:mainline-alpine
$ docker run -it --rm nginx:mainline-alpine sh
/ # apk update
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
v3.14.1-56-g3187897920 [https://dl-cdn.alpinelinux.org/alpine/v3.14/main]
v3.14.1-55-gd2a809b1f4 [https://dl-cdn.alpinelinux.org/alpine/v3.14/community]
OK: 14939 distinct packages available
/ # apk upgrade --simulate
OK: 25 MiB in 42 packages
/ # 
```

**Comment by thresheek** [comment]
CVE-2021-36159 is fixed in apk-tools-2.12.7-r0
CVE-2021-22922 is fixed in libcurl-7.78.0-r0
CVE-2021-22923 is fixed in libcurl-7.78.0-r0
CVE-2021-22924 is fixed in libcurl-7.78.0-r0
CVE-2021-22925 is fixed in libcurl-7.78.0-r0

All of mentioned packages/libraries are in the current `nginx:mainline-alpine`, so this is fixed.
